libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -334,7 +334,11 @@ ASTPointer<IdentifierPath> ASTJsonImporter::createIdentifierPath(Json::Value con
 		astAssert(!s.empty(), "Expected non-empty string for IdentifierPath element.");
 		namePath.emplace_back(s);
 	}
-	return createASTNode<IdentifierPath>(_node, namePath);
+	return createASTNode<IdentifierPath>(
+		_node,
+		namePath,
+		vector<SourceLocation>{ /* Data not exported, create dummy array. */ }
```

Comment:
I just suggested an solAssert() above, I think in that case, we should change that assert then to either match explicitly the counts or have zero source locations.